doc/api/stream.md

```diff
@@ -1722,6 +1722,141 @@ const cleanup = finished(rs, (err) => {
   // ...
 });
 ```
+#### Class: `stream.Body`
```

Comment:
While it makes things a bit more complicated, I would generally prefer to separate out the `Body` mixin pieces here from the Node.js specific additional APIs. Or, if not that, let's not call this `Body` to avoid confusion. `stream.Consumers` perhaps?

Further, I think an API with static methods similar to would be nicer here... it gives us more options for wrapping the methods in various API specific ways.

```mjs
const ab = await stream.consumers.arrayBuffer(readable);

// ...

const ab = await stream.consumers.blob(stream.compose(async function()* { yield 'hello'; }));
```